src/aiai_eval/task_configs.py

```diff
@@ -159,29 +162,11 @@ def get_all_dataset_tasks() -> Dict[str, DatasetTask]:
 )
 
 
-QA = DatasetTask(
-    name="qa",
-    supertask="question-answering",
-    metrics=[
-        MetricConfig(
-            name="em",
-            pretty_name="Exact Match",
-            huggingface_id="exact_match",
-            results_key="exact_match",
-        ),
-        MetricConfig(
-            name="f1",
-            pretty_name="F1-score of the positive class",
-            huggingface_id="f1",
-            results_key="f1",
-        ),
-    ],
-    labels=[],
-)
-
-
 SENT = DatasetTask(
     name="sent",
+    dataset_name="angry-tweets",
+    pretty_dataset_name="the truncated version of AngryTweets",
+    huggingface_id="ScandEval/angry-tweets-mini",
```

Comment:
Same as above with DaNE: We should use the official version of AngryTweets (https://huggingface.co/datasets/DDSC/angry-tweets)